005-RockPaperScissorsBoom/Coach/02-RunTheApp.md

```diff
@@ -0,0 +1,84 @@
+# Challenge 2 - Run the app
+
+[< Previous Challenge](01-Setup.md) - **[Home](README.md)** - [Next Challenge >](03-MoveToAzureSql.md)
+
+## Prerequisites
```

Comment:
Please remove all the Prerequisites sections from your guides. They are redundant since it's implied that What The Hacks' challenges are sequential and build upon each other.